qubes-rpc/qvm-actions.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Check if at least two arguments are provided: actions + file(s)
+if [ "$#" -le 1 ]; then
+    echo "Not enough arguments provided. Aborting..."
+fi
+
+# Action
+action="$1"
+
+shift
+
+# File(s)
+IFS='
+'
+files="$*"
+
+# copy and move handle a list of files where other actions don't
+case "$action" in
+    copy)
+        #shellcheck disable=SC2016
+        qvm-copy-to-vm '$default' "$@" | zenity --notification --text="Copying files..." --timeout 3
+        ;;
+    move)
+        #shellcheck disable=SC2016
+        qvm-move-to-vm '$default' "$@" | zenity --notification --text="Moving files..." --timeout 3
+        ;;
+    img)
+        for file in $files
```

Comment:
"$@" here should do here too (including proper whitespaces handling).

Reply:
You want me to drop the $files and replace it by $@? I can do it but with $* to take into account the newline IFS. I tried $@ and it does as usual, it splits when there is spaces in filenames.